tests/shouldwork/Signal/DynamicClocks.hs

```diff
@@ -0,0 +1,89 @@
+{-|
+Copyright  :  (C) 2022, Google Inc.
+License    :  BSD2 (see the file LICENSE)
+Maintainer :  QBayLogic B.V. <devops@qbaylogic.com>
+-}
+{-# LANGUAGE NumericUnderscores #-}
```

Comment:
`NumericUnderscores` is actually enabled by default.

Also, this module generates a bunch of warnings when compiled with `-Wall`. My personal preference is if it doesn't do that :-). Do with it what you will.

Reply:
Ah yeah, we should force that on CI.

Reply:
Ah wait, I thought this was about `unittests`. Forcing this check on CI would be quite the task. I've fixed the warnings for `DynamicClocks.hs` only.